classes/YARPP_Widget.php

```diff
@@ -31,34 +31,14 @@ public function widget($args, $instance) {
 
 		$title = apply_filters('widget_title', $instance['title']);
         $output = $before_widget;
-        if ($instance['use_pro']) {
-            if((isset($yarpp->yarppPro['active']) && $yarpp->yarppPro['active']) &&
-               (isset($yarpp->yarppPro['aid']) && isset($yarpp->yarppPro['v']))  &&
-               ($yarpp->yarppPro['aid'] && $yarpp->yarppPro['v'])) {
-
-                $aid  = $yarpp->yarppPro['aid'];
-                $v    = $yarpp->yarppPro['v'];
-                $dpid = $instance['pro_dpid'];
-                $ru   = 'http://'.$_SERVER['HTTP_HOST'].$_SERVER['REQUEST_URI'];
-
-                /* TODO: Put this on a template */
-                $output .=
-                    "\n".
-                    '<div id="adkengage_ssp_div"></div>'.
-                    '<script type="text/javascript" '.
-                    'src="http://adkengage.com/pshandler.js?aid='.$aid.'&v='.$v.'&dpid='.$dpid.'&ru='.$ru.'">'.
-                    '</script>'.
-                    "\n";
-            }
-        } else {
-            if (!$instance['template']) {
-                $output .= $before_title;
-                $output .= $title;
-                $output .= $after_title;
-            }
-            $instance['domain'] = 'widget';
-            $output .= $yarpp->display_related(null, $instance, false);
```

Comment:
this obviously needed to go, it was loading external JS which I'm pretty sure was fetching posts from other blogs, and it required payment to enable.